tasks/uglify.js

```diff
@@ -19,7 +20,7 @@ function relativePath(file1, file2) {
   var file2Dirname = path.dirname(file2);
 
   if (file1Dirname !== file2Dirname) {
-    return path.relative(file1Dirname, file2Dirname) + path.sep;
+    return path.relative(file1Dirname, file2Dirname);
```

Comment:
Thanks for the explanation.

We should be using `path.posix` methods when we drop node.js 0.10 support but that's for the future.